Reinstate G Suite deployment docs

lguychard opened the pull request:
Following https://github.com/sourcegraph/sourcegraph/pull/5894, and the deployment of v19.10.9.1226 of the browser extension, which supports setting the Sourcegraph URL through managed storage, this PR reinstates the G Suite deployment docs, with updated installation instructions.

dadlerj commented on doc/integration/google_gsuite.md:
```diff
@@ -0,0 +1,34 @@
+
+## Automatically install with G Suite
+
+Using the Google Admin Console, you can [force install](https://support.google.com/chrome/a/answer/6306504?hl=en) the Sourcegraph browser extension for all users in your organization. To do so:
```
```suggestion
Using the Google Admin Console, you can [automatically install](https://support.google.com/chrome/a/answer/6306504?hl=en) the Sourcegraph browser extension for all users in your organization. To do so:
```

dadlerj commented on doc/integration/google_gsuite.md:
```diff
@@ -0,0 +1,34 @@
+
+The Sourcegraph browser extension will then be rolled out to all managed browsers in your organization, preconfigured to point to your self-hosted Sourcegraph instance. The roll-out may take some time.
+
+Users will need to be logged in to your self-hosted Sourcegraph instance for the browser extension to provide code intelligence.
```
```suggestion
Users will need to have valid accounts and be signed in to your Sourcegraph instance for the browser extension to provide code intelligence.
```

lguychard commented:
Thanks for the feedback @dadlerj!

Added introductory paragraphs on security/privacy.

> is it possible to use G Suite to turn it on for just some people?

Yes, you can choose to deploy it to any organizational unit in your G Suite org. I've changed the wording to make this clear.

dadlerj commented on doc/integration/google_gsuite.md:
```diff
@@ -0,0 +1,42 @@
+
+<img src="img/disable_extension.png" width="400">
+
+By default, the browser extension only has access to github.com. Access to additional sites is granted by the user, on a site-by-site basis.
```
Is this true even for G Suite deployments? That's actually sort of annoying haha. There's no way for an admin to configure this?

dadlerj replied:
I.e., would each member of the team have to click on the icon to grant access to the code host, if it wasn't github.com?

lguychard replied:
Yes, this is annoying. I found no way for admins of directly configuring permissions, and the display of the permissions popup has to be triggered by user actions (so just setting an array of `codeHostUrls` through managed storage and calling `permissions.request()` for these URLs wouldn't work).

We could work around this by hinting to the user that he should open the popup (eg. by badging the icon) to grant additional permissions if code host URLs have been set through managed storage.

dadlerj commented on doc/integration/google_gsuite.md:
```diff
@@ -0,0 +1,42 @@
+
+You can install and preconfigure the Sourcegraph Chrome extension for all members of a G Suite organization or organizational unit, so that they can get code intelligence on files and diffs.
+
+Users can disable Sourcegraph at any time by using the toggle in the extension settings:
```
Sorry, but reading this now, this feels a bit out of place... Can we move this sentence and the screenshot to the very bottom instead?

lguychard replied:
Done.

dadlerj commented:
Thanks. Let's merge for now so I can send to https://app.hubspot.com/contacts/2762526/company/430823645, and I've filed https://github.com/sourcegraph/sourcegraph/issues/new?assignees=&labels=&template=question.md&title= for getting a review from Ryan (not urgent)!
